app/controllers/rents_controller.rb

```diff
@@ -41,29 +40,23 @@ def update
 
   def destroy
     @rent.destroy
-    redirect_to :bil, notice: alert_destroy(@rent)
+    redirect_to :rents, notice: alert_destroy(Rent)
   end
 
   # Index page available to logged in users.
   def index
-    if current_user.present?
-      @rents = current_user.rents.order('d_from desc')
-      render action: 'index'
-    else
-      redirect_to(action: 'main')
-    end
+    @rents = current_user.rents.order("d_from desc")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.